Userland/Libraries/LibCore/File.cpp

```diff
@@ -572,9 +572,12 @@ Optional<String> File::resolve_executable_from_environment(StringView filename)
     }
 
     auto const* path_str = getenv("PATH");
-    StringView path { path_str, strlen(path_str) };
-
-    if (path.is_empty())
+    StringView path;
+    if (path_str) {
+        path = { path_str, strlen(path_str) };
+        if (path.is_empty())
+            path = DEFAULT_PATH_SV;
+    } else
```

Comment:
In general, please put curly braces after the `else` statement when the `if` statement also has braces.

But this can be made simpler instead of having two paths that set `DEFAULT_PATH_SV`:
```suggestion
    StringView path;
    if (path_str)
        path = { path_str, strlen(path_str) };
    if (path.is_empty())
```

Reply:
Also for the commit title, let's not just use `...` to make it shorter, we can be a bit more descriptive:
`LibCore: Handle resolving executable paths even when PATH is null`